content/pages/download.md

```diff
@@ -105,11 +109,9 @@ versions:
       - slug: fedora
         os: Fedora
         text: |
-          An [RPM package](https://admin.rpmfusion.org/pkgdb/package/free/mixxx/) for installation is available in the [RPM Fusion](https://rpmfusion.org/) repositories.
-
-          Please refer to [RPM Fusion's instructions](https://rpmfusion.org/Configuration) on how to enable the repositories on your system. Mixxx only requires enabling the free repository; the nonfree repository is not necessary for Mixxx.
+          Mixxx 2.2 is no longer available as a Fedora package. Refer to the instructions for installing Mixxx 2.3 beta instead.
 
-          RPM Fusion builds are maintained by the Mixxx development team. We support the next, the current, and selected previous Fedora release(s) if possible.
+          WARNING: Do not use GNOME Software to install Mixxx. That installs an unofficial package from Flathub which does not work with PipeWire yet. This setup is not supported by the Mixxx development team.
```

Comment:
This sentence is exaggerating: "By default it installs an unofficial package from Flathub ..." (see also below)

Reply:
How is this exaggerating?

Reply:
Because it does **not** install the Flathub package unconditionally. The text gives the impression that I don't have any choice. But it is just a default that can be changed if you want.

Reply:
Oh, I misunderstood. I'll correct it.

Reply:
Let's play fair ;)

Reply:
I corrected the warning.